config/md5

```diff
@@ -1738,6 +1738,7 @@ ruby-2.7.4.tar.bz2=52705d799ed851dd3bfd5634265cde46
 ruby-2.7.5.tar.bz2=d97d4377eee18b0e790510c3afed648c
 ruby-2.7.6.tar.bz2=1b6efacb9f129d0253d8a9faa9c21f99
 ruby-2.7.7.tar.bz2=63c55e75150251d3ba092b662138e4af
+ruby-2.7.8.tar.bz2=b2d59ce994ae3d9715b15f842b2f9751
```

Comment:
Incorrect, should be `f44dedf51fdb7441a7dba592d6a4a14d`
Checked [archive from origin](https://cache.ruby-lang.org/pub/ruby/2.7/ruby-2.7.8.tar.bz2) with sha256 cross-check from [blog post](https://www.ruby-lang.org/en/news/2023/03/30/ruby-2-7-8-released/)

Reply:
`b2d59ce994ae3d9715b15f842b2f9751` is from tar.gz file

Reply:
#5346

Reply:
@hurricup can you check please?